### PR TITLE
insertion de l'image dans la show

### PR DESCRIPTION
**Pull request by ChrisVillaescusa**

### CODE REVIEW

**Comment by volchan** [comment]
i don't understand French sorry

File: app/views/gears/show.html.erb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <div class="container">
   <div>
-    <!-- Insérer la map Ici avec l'ID ' map-->
+    <% if @gear.photo? %>
+    <%= cl_image_tag @gear.photo.path, width: 200, height: 300 %>
```
Comment (volchan) [inline app/views/gears/show.html.erb:4]: indentation !!!!

Comment (ChrisVillaescusa) [reply app/views/gears/show.html.erb:4]: Indentation changed .
And french is stanslating

**Comment by ChrisVillaescusa** [comment]
translating

**Pull request merged**